script.js

```diff
@@ -0,0 +1,89 @@
+let playerScore = 0;
+let computerScore = 0;
+let playerChoice;
+
+function getComputerChoice() {
+  let computerChoice = Math.floor(Math.random() * 3);
+  if (computerChoice === 0) {
+    computerChoice = "rock";
+  } else if (computerChoice === 1) {
+    computerChoice = "paper";
+  } else {
+    computerChoice = "scissors";
+  }
+  return computerChoice;
+}
+
+const buttons = document.querySelectorAll(".buttons button");
+buttons.forEach((buttons) => buttons.addEventListener("click", processClick));
+
+function processClick() {
+  playerChoice = this.id;
+  playRound();
+}
+
+const reset = document.querySelector(".reset-button");
+reset.addEventListener("click", resetGame);
+
+function resetGame() {
+  playerScore = 0;
+  computerScore = 0;
+  playerChoice = null;
+  document.getElementById("c-score").innerHTML = computerScore;
+  document.getElementById("p-score").innerHTML = playerScore;
+  document.getElementById(
+    "results-text"
+  ).innerHTML = `Make your choice to start the game.`;
+  buttons.forEach((button) => {
+    button.addEventListener("click", processClick);
+    button.disabled = false;
+  });
+}
+
+function playRound() {
+  if (!playerChoice) {
+    return;
+  }
+  let computerChoice = getComputerChoice();
```

Comment:
this computerChoice is not overridden anywhere in playRound so it can safely be set to const.